artifacts/deploy/karmada-search.yaml

```diff
@@ -77,7 +77,8 @@ metadata:
     apiserver: "true"
 spec:
   ports:
-    - port: 443
+    - appProtocol: https
+      port: 443
```

Comment:
Karmada has different installation methods, and the directories `artifact/deploy` and `charts/karmada/templates` correspond to two different installation methods. Should the configurations of different installation methods be the same? If so, you can refer to https://github.com/karmada-io/karmada/issues/4191 to find the configuration files for different installation methods.

Reply:
`appProtocol` is well supported field regardless of the install method. shouldn't these two be in sync as much as possible?

Reply:
Yes, they should be in sync as much as possible. So you may need to make the same changes in different installation methods.